client/src/containers/projects/custom-project/mock-data.ts

```diff
@@ -0,0 +1,194 @@
+const tooltip = {
```

Comment:
Leave it for now, but this tooltip will probably go in a `constants/tooltip-info` file where we will host all the tooltips of the application there.